frontend/src/components/modals/AddTaskModal.module.scss

```
@@ -0,0 +1,4 @@
+textarea {
+    min-height: 20em !important;
```

Since the user can resize the area, I would add 20em as the default height with a possibility to go smaller. Not that users need a smaller field, it resizing just feels more fun (if that makes at all sense).

As a side note there the tab indent should be two spaces. Do you have automatic code formatting (with prettier) enabled on your editor? Let me know if you want setup help
```suggestion
  min-height: 5em !important;
  height: 20em;
```

---

It makes sense, I committed the changes.
I have a formatter (Prettier) but forgot to run it before committing.